apps/total-typescript/src/video/exercise-overlay.tsx

```diff
@@ -125,20 +128,84 @@ const ExerciseOverlay = () => {
     type: lesson._type,
   })
   const {github} = module
+  const {isSafari, isFirefox} = useDeviceDetect()
+  const isStackblitzCompatibleBrowser = !(isSafari || isFirefox)
+
+  const {exerciseGitHubUrl} = getExerciseGitHubUrl({stackblitz, module})
 
   return status !== 'loading' ? (
     <div className=" bg-black/30 ">
       {stackblitz ? (
         <>
           <div className="flex w-full items-center justify-between p-3 pl-5 font-medium sm:text-lg">
-            <div>Now it's your turn! Try solving this exercise.</div>
+            <div className="flex flex-col">
+              <div>Now it's your turn! Try solving this exercise.</div>
+            </div>
             <div className="flex justify-center gap-2">
               <Actions />
             </div>
           </div>
           <div className="relative hidden h-[500px] w-full sm:block xl:h-[750px]">
             <StackBlitzIframe exercise={lesson} module={module} />
           </div>
+          {!isStackblitzCompatibleBrowser && (
```

Comment:
I dropped this below the frame since it felt bulky up top and also secondary "FYI" type info.